API.md

```diff
@@ -575,7 +581,7 @@ name [string](https://godoc.org/builtin#string)
 star_count [int](https://godoc.org/builtin#int)
 ### <a name="InfoGraphStatus"></a>type InfoGraphStatus
 
-InfoGraphStatus describes the detailed status of the graphc
+InfoGraphStatus describes the detailed status of the graph
```

Comment:
What is a graph?

Reply:
We could call it "graph driver" to match Docker, but I'd personally prefer calling it something like StorageDriver

Reply:
Sounds much better to me.  Graph is a very internal name.

Reply:
storage driver it is